README.md

```diff
@@ -4,11 +4,15 @@
 [![coveralls][coveralls-badge]][coveralls-url]
 [![snyk][snyk-badge]][snyk-url]
 
-Udaru is a policy based authorization module that can be used to add permissions to 'actions' and 
-'resources'. Udaru supports 'organizations', 'teams', and 'users'; policies can be created for 
-each. Udaru can be used as a stand-alone module , or as a stand-alone server or 
+Udaru is a policy based authorization module that can be used to add permissions to 'actions' and
+'resources'. Udaru supports 'organizations', 'teams', and 'users'; policies can be created for
+each. Udaru can be used as a stand-alone module , or as a stand-alone server or
 Hapi plugin. This repository contains the code for all three running configurations.
 
+A quick [Authorization Introduction][] on how the Udaru authorization system is structured and the terms and elements used to define the authorization system.
```

Comment:
`You can find a detailed explanation on how the Udaru authorization system is structured and the terms and elements used in the [Authorization Introduction](#) section`?